cmd/oras/internal/display/metadata/json/pull.go

```diff
@@ -30,11 +30,8 @@ type PullHandler struct {
 	path   string
 	pulled model.Pulled
 	out    io.Writer
-}
-
-// OnLayerSkipped implements metadata.PullHandler.
-func (ph *PullHandler) OnLayerSkipped(ocispec.Descriptor) error {
-	return nil
+	target *option.Target
+	desc   ocispec.Descriptor
```

Comment:
Does `desc` refers to the root node? If so, should we name it "root" to be consistent with other handlers?

Reply:
renamed.